rai/node/bootstrap.cpp

```diff
@@ -8,6 +8,7 @@
 constexpr double bootstrap_connection_scale_target = 50000.0;
 constexpr double bootstrap_connection_warmup_time = 5.0;
 constexpr double bootstrap_minimum_block_rate = 10.0;
+constexpr double bootstrap_minimum_frontier_rate = 1000.0;
```

Comment:
Please change these identifiers to include the units for the quantities.

e.g.

`bootstrap_minimum_frontier_rate` -> `bootstrap_minimum_frontier_rate_blocks_per_sec`
`bootstrap_connection_warmup_time` -> `bootstrap_connection_warmup_time_sec`

Reply:
Done